spec/Phpro/SoapClient/BridgeBundle/Type/SoapCallSpec.php

```diff
@@ -74,6 +95,11 @@ function it_should_parse_the_debug_array(
             ],
         ];
         $client->debugLastSoapRequest()->willReturn($debugData);
+
+        $requestEvent->getRequest()->shouldBeCalled();
```

Comment:
These should also return something instead of validating wether the method was called.